src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] arrayOfBalls = new Ball[3];
```

Comment:
better create constant `NUMBER_OF_BALLS` for number `3`
read more about conatants here - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names